ios/Sources/Component/NavigationBar/NavigationBarConfigurator.swift

```diff
@@ -0,0 +1,58 @@
+import Styleguide
+import SwiftUI
+
+public extension View {
+    func navigationBarColor(backgroundColor: Color, titleColor: Color) -> some View {
+        modifier(NavigationBarConfigurator(backgroundColor: backgroundColor, titleColor: titleColor))
+    }
+}
+
+fileprivate struct NavigationBarConfigurator: ViewModifier {
+    
+    private let newBackgroundColor: Color
+    private let newTitleColor: Color
+    
+    private let previousBackgroundColor: Color?
+    private let previousTitleColor: Color?
+    
+    init(backgroundColor: Color, titleColor: Color) {
+        
+        self.newBackgroundColor = backgroundColor
+        self.newTitleColor = titleColor
+        
+        if let previousTitleColor = UINavigationBar.appearance().standardAppearance.titleTextAttributes[.foregroundColor] as? UIColor {
+            self.previousTitleColor = Color(previousTitleColor)
+        } else {
+            self.previousTitleColor = nil
+        }
```

Comment:
I think this is okay 😄 
```suggestion
        if let previousTitleColor = UINavigationBar.appearance().standardAppearance.titleTextAttributes[.foregroundColor] as? UIColor {
            self.previousTitleColor = Color(previousTitleColor)
        }
```